functions/functions/models/schemas.py

```
@@ -96,3 +96,48 @@
+
+
+class CreateSyncProfileInput(BaseModel):
+    title: str = Field(
+        ...,
+        description="User-defined title for the sync profile.",
+        min_length=3,
+        max_length=50,
+        examples=["My University Schedule", "Master IF Semestre 1"],
+    )
+
+    scheduleSource: ScheduleSource
```

Ruff (0.8.2)

Variable `scheduleSource` in class scope should not be mixedCase

(N815)